src/components/modules/toolbar/blockSettings.ts

```diff
@@ -170,6 +170,10 @@ export default class BlockSettings extends Module<BlockSettingsNodes> {
     }
 
     this.selection.clearSaved();
+    /**
+     * Remove highlighted content of a Block we are working with
+     */
+    this.Editor.BlockManager.currentBlock.selected = false;
```

Comment:
When clicking the outside of Editor.js, the following errors are occured: 
```
Uncaught TypeError: Cannot set property 'selected' of undefined
```

```suggestion
    if (this.Editor.BlockManager.currentBlock) {
      this.Editor.BlockManager.currentBlock.selected = false;
    }
```

Reply:
Done